pom.xml

```diff
@@ -120,6 +120,7 @@
             </plugin>
             <plugin>
                 <artifactId>maven-surefire-plugin</artifactId>
+                <version>2.18.1</version>
```

Comment:
In other sources MMX manages it's set in the oss-parent. I'll chat with the other devs to see if there's a reason it was removed from there.